x/auction/types/params.go

```diff
@@ -51,3 +74,39 @@ func validatePriceMaxBlockAge(i interface{}) error {
 
 	return nil
 }
+
+func validateMinimumBidInUsomm(i interface{}) error {
+	_, ok := i.(uint64)
+	if !ok {
+		return sdkerrors.Wrapf(ErrMinimumBidParam, "invalid minimum bid in usomm parameter type: %T", i)
+	}
+
+	return nil
+}
+
+func validateAuctionMaxBlockAge(i interface{}) error {
+	auctionMaxBlockAge, ok := i.(uint64)
+	if !ok {
+		return sdkerrors.Wrapf(ErrInvalidAuctionMaxBlockAgeParam, "invalid auction max block age parameter type: %T", i)
+	}
+
+	if auctionMaxBlockAge == 0 {
```

Comment:
Arguably wouldn't an option to not prune be desirable?

Reply:
Yeah that'd make sense, will update

Reply:
Updated.